### ISSUE TITLE
Alts + Premium

### ISSUE THREAD

**Issue by 6BytesIn** [ImpactDevelopment/ImpactClient#201]
Keep premium after changing alts

**Comment by hugohindi** [comment]
Impact rats your computer anyway so it would be easy to give premium to every alt on a specific computer

**Comment by 6BytesIn** [comment]
:o.. b-b-rady ccan.. see my henta-i?

**Comment by SuperOP535** [comment]
breadman can see everything in you pc

**Comment by zPeanut** [comment]
what even is this conversation

**Comment by SuperOP535** [comment]
uhh whats this suggestion? some kind of "free premium" (which is not possible) or what the hell?

**Comment by ZeroMemes** [comment]
Premium should be HWID based, not UUID based like it currently is. @SuperOP535

**Comment by SuperOP535** [comment]
ohh I understand. I thought he was meaning some kind of Minecraft premium account lol

**Comment by LeafHacker** [comment]
HWID is just as bad as UUID, think about changing computer or having
multiple computers.

One possible hack would be to cache   premium and return whether or not it
has been cached yet.

Effectively if you login to a premium uuid, premium would stay active until
you quit MC, even if you log in again to another account.

**Comment by ZeroMemes** [comment]
@LeafHacker True, some sort of key file should exist. I'll make premium stay active as long as you're in a valid alt for the time being. Later a separate issue can be opened for non-UUID based premium.
